purchase_transport_multi_address/test/test_propagate_address.yml

```diff
@@ -0,0 +1,29 @@
+-
+  I select for this file a user that is not admin
+-
+  !context
+  uid: 'res_users_giulia'
+-
+  I create a quotation with a dropshipping line.
+-
+  !record {model: sale.order, id: so_4}:
+    partner_id: base.res_partner_3
+    order_line:
+      - product_id: product.product_product_7
+        product_uom_qty: 8
+        route_id: route_drop_shipping
+-
+  I confirm the sale order, run the scheduler, and check that the address of
+  the sale order has been propagated to the automatically generated purchase
+  order.
+-
+  !python {model: sale.order, id: so_4}: |
+    from nose.tools import *
+
+    self.action_button_confirm()
+    self.env['procurement.order'].run_scheduler()
```

Comment:
@nbessi AFAIK when called like that `run_scheduler` uses the same thread and cursor, so it is synchronous, which is correct for a test.